devel/libprotozero/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                                    1.0
+PortGroup           cmake                     1.1
+PortGroup           github                    1.0
+
+name                libprotozero
+
+github.setup        mapbox protozero 1.7.0 v
```

Comment:
the github PG sets the `name` of the port to the project name on GitHub - so in this case the name would be `protozero`. You're overriding that by explicitly setting the name to `libprotozero`; in itself is perfectly fine, but is there a reason to change the name of this port? It seem that many other [package managers](https://repology.org/project/protozero/versions) do install this as `protozero`.

Reply:
I'll rename to protozero.  Thanks for the link to Repology, a very useful resource I wasn't aware of.